### PR TITLE
style other innovation communities section

### PR DESCRIPTION
**Pull request by Mel-Leusch**
[Preview](https://federalist-cdebd6cd-602d-4428-9db1-d140287d165e.app.cloud.gov/preview/gsa/citizenscience.gov/staging/toolkit/#)

### CODE REVIEW

**Comment by greensteph** [comment]
Move section to Communities Page

- rename Other..... to Crowdsourcing Communities 

- add code.gov

- remove socialgov 

Use logos/wordmarks



- leave challenge COP in

File: pages/toolkit/toolkit-home.md [hunk 1]
```diff
@@ -75,19 +75,18 @@ banner-button-link: https://ccsinventory.wilsoncenter.org/
...
-			<li><a href="https://www.digitalgov.gov/communities/" target="blank">DigitalGov</a></li>
-			<li><a href="https://www.data.gov/" target="blank">Data.gov</a></li>
-			<li><a href="https://www.digitalgov.gov/communities/social-media/" target="blank">SocialMedia CoP</a></li>
-		</ul>
+<section class="usa-grid usa-section innovation-communities"> 
+    <div class="usa-width-one-whole innovation-communities">
+		<h2 class="innovation-communities">Crowdsourcing Communities</h2>
+			<ul class="innovation-communities">
+				<li class="innovation-communities"><a href="https://www.digitalgov.gov/communities/challenges-prizes/" target="blank"><img src="{{ site.baseurl }}\assets\img\toolkit-images\logos\challenge-logo.png" alt="Challenge.gov logo" class="innovation-communities-challenge"></a></li>
+				<li class="innovation-communities"><a href="https://code.gov/" target="blank"><img src="{{ site.baseurl }}\assets\img\toolkit-images\logos\code-logo.svg" alt-="Code.gov logo" class="innovation-communities"></a></li>
```
Comment (greensteph) [inline pages/toolkit/toolkit-home.md]: Fix `alt-="Code.gov logo"`

File: pages/toolkit/toolkit-home.md [hunk 2]
```diff
@@ -75,19 +75,18 @@ banner-button-link: https://ccsinventory.wilsoncenter.org/
...
+				<li class="innovation-communities"><a href="https://www.data.gov/" target="blank"><img src="{{ site.baseurl }}\assets\img\toolkit-images\logos\data-logo.png" al="Data.gov logo" class="innovation-communities"></a></li>
```
Comment (greensteph) [inline pages/toolkit/toolkit-home.md]: Fix `al="Data.gov logo"`

**Comment by Mel-Leusch** [comment]
Add other back to the title
add more padding between logos
add vertical columns between logos
take the width out

**Pull request merged**